### ISSUE TITLE
OMDB API requires an API Key

### ISSUE THREAD

**Issue by jkoech** [hareshkh/OMDb-Android#1]
Doesn't work since OMDB now requires an API key.

**Comment by hareshkh** [comment]
This was just a demo project and I am no longer working on it. Do you want to improve the code and make the appropriate changes?

**Comment by jkoech** [comment]
Yes, I just want to address a few edge cases. Will work on it and send a PR.
